### ISSUE TITLE
Failed R installation

### ISSUE THREAD

**Issue by devmon** [CABSEL/CALISTA#1]
Hello, I am trying to install calista in R and receive the following error:

```
* installing *source* package 'calista' ...
** using staged installation
** libs

*** arch - i386
C:/RBuildTools/3.5/mingw_32/bin/gfortran   "-fno-stack-protector"   -O3  -mtune=generic -c  greedy.f95 -o greedy.o
greedy.f95:714.17:

real*8 cell_prob(nvars,expected_clusters), bbb(q,expected_clusters)
                 1
Error: Variable 'nvars' cannot appear in the expression at (1)
make: *** [C:/PROGRA~1/R/R-36~1.0/etc/i386/Makeconf:234: greedy.o] Error 1
ERROR: compilation failed for package 'calista'
```

I am using R ver 3.6 and RStudio ver 1.2.

Thanks!

**Comment by nanp** [comment]
Hello @devmon,

thank you for reporting the issue. I think there's a compatibility problem with fortran compiler in Windows. Can you please try to type **Sys.setenv(PKG_FCFLAGS = "")**  in R before installing it?(Based on the following [post](https://stackoverflow.com/questions/31638934/r-package-with-fortran-module-on-windows-undefined-reference-to-stack-chk-fa) )

**Comment by nanp** [comment]
an alternative option is to install up to date builds of GCC and gfortran from Cygwin. This has been shown to solve the issue on Windows 10.

**Comment by devmon** [comment]
I tried the Sys.setenv(PKG_FCFLAGS = "") and it did not work. I have the most up to date builds of GCC and gfortran from Cygwin and it failed with the same error.

I followed the error in the greedy.f95 file at line 714 and noticed nvars was not declared in the function. The function was also not used so I commented it out. I forked the project and the build was successful. I ran one of the tutorials and it works!
